libs/shared/metrics/glean/src/lib/metrics.context.ts

```diff
@@ -33,7 +33,7 @@ export class MetricsContext {
     queryParams = queryParams || {};
     this.deviceId = queryParams['deviceId'];
     this.entrypoint = queryParams['entrypoint'];
-    this.flowId = queryParams['flowId'];
+    this.flowId = queryParams['flowId'] || queryParams['flow_id'];
```

Comment:
question: Should the other query params also be updated to support camel and pascal case?